woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1814,3 +1820,80 @@ def test_maintain_column_order_disordered_schema(sample_df):
     sample_df.ww.init(schema=scramble_df.ww.schema)
     assert all(sample_df.columns == column_order)
     assert all(sample_df.ww.types.index == column_order)
+
+
+def test_accessor_types(sample_df):
+    sample_df.ww.init()
+
+    returned_types = sample_df.ww.types
+    assert isinstance(returned_types, pd.DataFrame)
+    assert 'Physical Type' in returned_types.columns
+    assert 'Logical Type' in returned_types.columns
+    assert 'Semantic Tag(s)' in returned_types.columns
```

Comment:
If we want to make sure the column order is consistent, maybe we should check that the list of columns matches what we expect, instead of just checking that certain names are present.